CHANGELOG.rst

```diff
@@ -12,14 +12,30 @@ Changelog
 **New feature**
 
 - Expose the :func:`ndonnx.isdtype` function.
-
+- Custom dtypes can now override array creation functions:
+   - :func:`ndonnx.zeros`
+   - :func:`ndonnx.zeros_like`
+   - :func:`ndonnx.ones`
+   - :func:`ndonnx.ones_like`
+   - :func:`ndonnx.full`
+   - :func:`ndonnx.full_like`
+   - :func:`ndonnx.arange`
+   - :func:`ndonnx.arange`
+   - :func:`ndonnx.eye`
+   - :func:`ndonnx.tril`
+   - :func:`ndonnx.triu`
+   - :func:`ndonnx.linspace`
+- Custom dtypes can now provide implementations for :func:`ndonnx.where`.
+- The :class:`ndonnx._experimental.UniformShapeOperations` now provides implementations of shape operations that are generic across all data types where each constituent field has the same shape (that of the overall array).
 
 **Other changes**
 
 - Fixed various deprecation warnings.
+- Invoking a function using arrays with dtypes lacking a corresponding implementation now raise a :class:`UnsupportedOperationError`.
```

Comment:
```suggestion
- Invoking a function using arrays of data types that lack a corresponding implementation now raises an :class:`UnsupportedOperationError`.
```

Reply:
"arrays of data types" should be "arrays with data types" in my opinion but I took the second half of the suggestion.